src/Windowing/Silk.NET.Windowing.Common/Internals/WindowImplementationBase.cs

```diff
@@ -0,0 +1,176 @@
+﻿// This file is part of Silk.NET.
+// 
+// You may modify and distribute Silk.NET under the terms
+// of the MIT license. See the LICENSE file for details.
+
+using System;
+using System.Drawing;
+using Silk.NET.Windowing.Structs;
+
+// No immediate urgency to document these as they're not intended for public consumption.
+#pragma warning disable 1591
+
+namespace Silk.NET.Windowing.Internals
+{
+    /// <summary>
+    /// Abstracts away common window functions to ease implementation of the windowing API.
+    /// </summary>
+    public abstract class WindowImplementationBase : ViewImplementationBase, IWindow
```

Comment:
This shouldn't be public (as anything in a `Internals` namespace with a notice "not intended for public consumption", so won't add any more comments)

Reply:
Same comment as above.

Reply:
Fixed.